Octokit.Tests.Integration/Helper.cs

```diff
@@ -50,9 +50,9 @@ public static class Helper
             return new Credentials(githubUsername, githubPassword);
         });
 
-        static readonly Lazy<Uri> _gitHubEnterpriseUrl = new Lazy<Uri>(() =>
+        static readonly Lazy<Uri> _customUrl = new Lazy<Uri>(() =>
         {
-            string uri = Environment.GetEnvironmentVariable("OCTOKIT_GITHUBENTERPRISEURL");
+            string uri = Environment.GetEnvironmentVariable("OCTOKIT_CUSTOMURL");
```

Comment:
So originally that was my "hack" to allow me to run regular integration tests against a GitHub enterprise instance.  But now that I've officially added an EnterpriseHelper class it isnt appropriate to refer to this (in normal helper) as Enterprise URL.  This item still remains so that someone COULD point "all" integration tests at another instance (eg enterprise) if they wanted, by setting the CustomURL.  But the difference is, that all the other paramters like username, password, apitoken etc would be coming from the "regular" settings.

Meanwhile for tests that explicitly require/target GitHub Enterprise, we use the EnterpriseHelper class which has it's own setup for username, password, api key and so on.  

Basically this means that I can have my workstation configured to run integration tests against github.com and enteprise integration tests against my enterprise instance... Plus if I ever need to run the "normal" integration tests against my enterprise instance, I can set the CUSTOM_URL property.

Hope that makes sense